ratis-server-api/src/main/java/org/apache/ratis/server/leader/FollowerInfo.java

```diff
@@ -52,6 +52,14 @@
   /** Set follower's snapshotIndex. */
   void setSnapshotIndex(long newSnapshotIndex);
 
+  /** Acknowledge that Follower attempted to install a snapshot. It does not guarantee that the installation was
+   * successful. This helps to determine whether Follower can come out of bootstrap process. */
+  void ackInstallSnapshotAttempt();
```

Comment:
Let's use a name similar to hasAttemptedToInstallSnapshot().  How about "setAttemptedToInstallSnapshot"?

Reply:
Done.